crates/revm/Cargo.toml

```diff
@@ -54,20 +54,22 @@ ethers-core = { version = "2.0", optional = true }
 # alloy-transport = { git = "https://github.com/alloy-rs/alloy.git", rev = "44b8a6d", optional = true, default-features = false }
 
 [dev-dependencies]
+alloy-provider = { git = "https://github.com/alloy-rs/alloy.git", rev = "44b8a6d", default-features = false, features = [
+    "reqwest",
+] }
 alloy-sol-types = { version = "0.7.0", default-features = false, features = [
     "std",
 ] }
+alloy-sol-macro = "0.7.0"
+# needed for enabling TLS to use HTTPS connections when testing alloy DB
+alloy-transport-http = { git = "https://github.com/alloy-rs/alloy.git", rev = "44b8a6d" }
```

Comment:
as Alloy is not in crates this would block the publishing of revm to crates. We can't use it.

Reply:
I believe publishing should still work because these are dev deps which cargo removes before publishing afaik

Reply:
I'll go ahead and undo this reordering of dependencies. Only the `rstest` line should be new

Reply:
my bad, didn't see it was in dev deps